Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/contactdiary/DiaryData.kt

```diff
@@ -18,40 +19,35 @@ object DiaryData {
     val DATA_ITEMS = listOf(
         ListItem.Data(
             R.drawable.ic_contact_diary_person_item,
-            "Max Mustermann",
-            null,
-            listOf(
-                R.string.contact_diary_person_encounter_duration_below_15_min,
-                R.string.contact_diary_person_encounter_mask_with,
-                R.string.contact_diary_person_encounter_environment_inside
-            ),
-            "Notizen notizen",
-            ListItem.Type.PERSON
+            "Rewe",
+            Duration.standardMinutes(30),
+            attributes = null,
+            circumstances = null,
+            ListItem.Type.LOCATION
```

Comment:
On the screenshot, the icon for a person is shown instead. please double check.

Reply:
Good eyes 👀 😀  => I changed the icon, see new screenshot: 

![ContactDiaryOverviewFragment](https://user-images.githubusercontent.com/10398034/109298361-c0e79600-7833-11eb-862e-65e03367a295.png)